images/ose-kubevirt-csi-driver-rhel8.yml

```diff
@@ -0,0 +1,30 @@
+content:
+  source:
+    dockerfile: Dockerfile.openshift
+    git:
+      branch:
+        target: release-{MAJOR}.{MINOR}
+      url: git@github.com:openshift-priv/kubevirt-csi-driver.git
+      web: https://github.com/openshift/kubevirt-csi-driver
+    ci_alignment:
+      streams_prs:
+        ci_build_root:
+          stream: rhel-8-golang-ci-build-root
+    pkg_managers:
+    - gomod
+enabled_repos:
+- rhel-8-baseos-rpms
+- rhel-8-appstream-rpms
+for_payload: false
+distgit:
+  component: ose-kubevirt-csi-driver-container
+from:
+  builder:
+  - stream: golang
+  member: openshift-enterprise-base
+name: openshift/ose-kubevirt-csi-driver
+owners:
```

Comment:
can you add me as an owner too please? dvossel@redhat.com

Reply:
got it